example/src/app.tsx

```diff
@@ -1,14 +1,12 @@
 import './wdyr'; // <--- must be first import
-import {Navigation} from 'react-native-navigation';
-// import {I18nManager} from 'react-native'; // <--- In order to test RTL
+import React from 'react';
+import {AppRegistry} from 'react-native';
 //@ts-expect-error
```

Comment:
I figure that if you include this file in tsconfig, you won't receive TS errors about this

Reply:
True, but I have other issues with LocaleConfig that I didn't want to deal with now. I'll see if I have time for that, I'll try to fix them